mitodl/mitxonline
# Adds currency code descriptions; makes sure invalid codes are removed

#### Pre-Flight checklist


#### What are the relevant tickets?

Fixes #886

#### What's this PR do?

Updates the exchange rate tasks to pull and store the currency descriptions alongside the current rates. Also adds some constants for disallowed currencies; these will not be stored in the table. 

#### How should this be manually tested?

Add an invalid currency or update an existing currency with some other value, then run the `update_exchange_rates` command. The changes you made should be removed.

The log output (if the log level is set appropriately; i.e. to `DEBUG`), you should see skip messages for currencies that have rates but are defined in `main/constants.py` as disallowed.
